model/distro/distro.go

```diff
@@ -593,6 +598,15 @@ func (d *Distro) GetRegionsList() []string {
 			})
 			continue
 		}
+		// this region isn't allowed to be spawned by admins
```

Comment:
this threw me for a loop.
"admins don't allow this region to be spawned" might have been clearer to me.